src/test/script_P2SH_tests.cpp

```diff
@@ -53,6 +54,7 @@ BOOST_FIXTURE_TEST_SUITE(script_P2SH_tests, BasicTestingSetup)
 BOOST_AUTO_TEST_CASE(sign)
 {
     LOCK(cs_main);
+    SelectPolicy("standard");
```

Comment:
Yes BasicTestingSetup sets the `standard` policy so there is no reason to do it in the individual test cases too.